tasks/cachebust.js

```diff
@@ -59,9 +60,11 @@ module.exports = function() {
             }));
             var newFilename = addFileHash(file, hash, opts.separator);
 
-            grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            if(opts.createCopies) {
+                grunt.file.copy(absPath, path.resolve(opts.baseDir, newFilename));
+            }
 
-            if(opts.deleteOriginals) {
+            if(opts.deleteOriginals && opts.createCopies) {
```

Comment:
I don't see the need for this addition. If someone is going to set `createCopies` to `false` and `deleteOriginals` to `true` then let hell rain down on them